Source/SwiftLintFramework/Rules/Lint/PrivateSwiftUIStatePropertyRule.swift

```diff
@@ -0,0 +1,99 @@
+import SwiftSyntax
+
+/// Rule to require that any state properties in SwiftUI be declared as private.
+/// State properties should only be accessible from inside the View's body, or from methods called by it
+struct PrivateSwiftUIStatePropertyRule: SwiftSyntaxRule, OptInRule, ConfigurationProviderRule {
+    var configuration = SeverityConfiguration(.warning)
+
+    static let description = RuleDescription(
+        identifier: "private_swiftui_state",
+        name: "Private SwiftUI @State properties",
+        description: "SwiftUI state properties should be private",
```

Comment:
Does that sound better?
```suggestion
        description: "SwiftUI's state properties should be private",
```